src/include/cell.i

```diff
@@ -799,10 +851,17 @@ restart:
             WT_RET(__wt_vunpack_uint(&p, end == NULL ? 0 : WT_PTRDIFF(end, p), &unpack->stop_txn));
             unpack->stop_txn += unpack->start_txn;
         }
+        if (LF_ISSET(WT_CELL_TS_DURABLE_STOP)) {
+            WT_RET(__wt_vunpack_uint(
+              &p, end == NULL ? 0 : WT_PTRDIFF(end, p), &unpack->durable_stop_ts));
+            unpack->durable_stop_ts += unpack->stop_ts;
+        }
         __cell_check_value_validity(
           session, unpack->start_ts, unpack->start_txn, unpack->stop_ts, unpack->stop_txn);
         break;
     }
+    if (F_ISSET(unpack, WT_CELL_UNPACK_PREPARE))
+        WT_RET(__cell_unexpected_prepare(session));
```

Comment:
@keithbostic, I want to draw attention to this latest change.  If the prepare bit is set, we'll return an error when unpacking.  This differs from the discussion we had during the design spec, where we agreed that prepared items would be "skipped".   I'm not sure that's easy to do, the unpacking functions are called all over the place, and it's quite a procedure to get the prepare flag out to where it's needed to make that decision.  To remind you, the prepare bit should not be set on a "properly" downgraded system (`WT_CONNECTION->close` will be called, and all prepared transactions aborted). It only comes up if someone recklessly starts up 4.2 on some 4.4 files caught in mid-flight.

Reply:
This is after a downgrade so my inclination is to simply ignore the bit.